detox/test/src/app.js

```diff
@@ -134,7 +134,11 @@ export default class example extends Component {
 
         {isIos && this.renderScreenNotifyingButton_iOS('Custom Keyboard', 'customKeyboard')}
 
-        {this.renderScreenButton('Element-Screenshots', Screens.ElementScreenshotScreen)}
+        <View style={{flexDirection: 'row', justifyContent: 'center'}}>
+          {this.renderScreenButton('Element-Screenshots', Screens.ElementScreenshotScreen)}
+          {this.renderInlineSeparator()}
+          {isAndroid && this.renderScreenButton('Canvas-Screenshots', Screens.CanvasScreenshotScreen)}
+        </View>
```

Comment:
just on android, because drawing skia requires no special case on ios, but theoretically we could run the test on ios as well, I guess